firestore/cloud-async-client/snippets.py

```diff
@@ -17,66 +17,57 @@
 
```

Comment:
I think if you remove the comma after the year if should be fine, or `Copyright 2020 Google LLC` should do.

Reply:
Done